osquery/tables/networking/windows/process_open_sockets.cpp

```diff
@@ -5,12 +5,36 @@
  *  This source code is licensed in accordance with the terms specified in
  *  the LICENSE file found in the root directory of this source tree.
  */
+#include <string>
```

Comment:
nitpick, do you mind keeping a blank newline between the header comment and the start of code?

Reply:
Sure. I'll make the change.